src/frontend/next-env.d.ts

```diff
@@ -1,5 +1,4 @@
 /// <reference types="next" />
-/// <reference types="next/types/global" />
```

Comment:
What's this about?

Reply:
a little confusing to tell from the docs but this [file is regenerated when you run `next build`](https://github.com/vercel/next.js/issues/26533), i think it removed that line because that next type is no longer relevant to the typescript compiler with the upgrade

Reply:
I just needed to docker-compose down then up the containers -- also ran a DB migration, but I don't think that matters -- and this line re-appeared in my `next-env.d.ts` file.
```
/// <reference types="next/types/global" />
```

Maybe we should re-commit it back in? Probably has no impact on how it runs since we build during the CI pipeline, but if it keeps showing up as a dirty file locally that will just be confusing to see.